newsletters/2025-05-21-newsletter .md

```diff
@@ -0,0 +1,51 @@
+# ZK Newsletter for 2025-05-21
+This release details...
+
+### Top picks
+
+### Reading materials 
+* [Valida ISA Spec, version 1.0: A zk-Optimized Instruction Set Architecture](https://arxiv.org/pdf/2505.08114) - academic text by Morgan Thomas, Mamy Ratsimbazafy, Marcin Bugaj, Lewis Revill, Carlo Modica, Sebastian Schmidt, Ventali Tan, Daniel Lubarov, Max Gillett and Wei Dai. (12.05.2025)
+* [Finite Field Arithmetic Optimization](https://medium.com/@CFrontier_Labs/finite-field-arithmetic-optimization-ab449202c776) - article by Frontier Labs. (18.05.2025)
+* [FairZK: A Scalable System to Prove Machine Learning Fairness in Zero-Knowledge](https://arxiv.org/pdf/2505.07997) - academic text by Tianyu Zhang, Shen Dong, O. Deniz Kose, Yanning Shen, Yupeng Zhang. (18.05.2025)
+* [Comparing Optimistic or ZK Rollup Technologies and Their Ideal Applications](https://medium.com/@Gateway.FM/comparing-optimistic-or-zk-rollup-technologies-and-their-ideal-applications-2c683a6f1288) - article by Gateway.FM. (19.05.2025)
+* [Native Rollups: Where they are, and where they are going](https://medium.com/l2beat/native-rollups-where-they-are-and-where-they-are-going-cb21eb103d46) - article by Luca Donno. (19.05.2025)
+* [Reed Solomon Fingerprint, Reed Solomon Encoding, Low Degree Extension and Lagrange Interpolation: First Steps from Maths to Magics [PART II]](https://medium.com/@0xlipefe/reed-solomon-fingerprint-reed-solomon-encoding-low-degree-extension-and-lagrange-interpolation-0f2a7d471da7) - article by Lipefe. (19.05.2025)
+
+### Watching materials
+* [Verifiable compute is eating the world - Presentation by Wyatt Benno, Core Contributor at NovaNet](https://www.youtube.com/watch?v=h7Bzeu__pos). (13.05.2025)
+* [Building a Bitcoin dApp- Broly Demo and Dev Libraries | Starknet](https://www.youtube.com/watch?v=xsh3q8d0LZw). (18.05.2025)
+* [Presentation by Daniel Kwak, LazAI: A Hidden Crisis in AI - The Data Alignment Problem](https://www.youtube.com/watch?v=prPEqRSbiBU). (19.05.2025)
+ 
+### Listening materials
+* [Zero Knowledge Podcast](https://zeroknowledge.fm/podcast/361/) episode - *Quantum Engineering with Jelena Vučković*. (14.05.2025)
```

Comment:
The name of this episode is "Implementing LatticeFold with Matthew and Albert from Nethermind". Correct it. Top pick